src/chatkit.ts

```diff
@@ -172,6 +172,11 @@ export interface GetReadCursorsForRoomOptions {
   roomId: string
 }
 
+export interface FetchMultipartMessageOptions {
+  roomId: string
+  messageId: string
```

Comment:
message ID should probably be a `number`. See for example `position` in `SetReadCursorOptions`.